.github/workflows/build.yaml

```
@@ -472,34 +464,102 @@ jobs:
+        uses: actions/download-artifact@v8
+        with:
+          name: Windows-app
+          path: .app
+
+      - name: Build and sign Velopack package
+        shell: bash
+        env:
+          PACK_ID: ${{ needs.build.outputs.velopack_pack_id }}
+          PACK_VERSION: ${{ needs.build.outputs.velopack_pack_version }}
+          PACK_TITLE: ${{ needs.build.outputs.velopack_pack_title }}
+          PACK_AUTHORS: ${{ needs.build.outputs.velopack_pack_authors }}
```

actionlint (1.7.12)

[error] property "velopack_pack_authors" is not defined in object type {imagename: string; velopack_exclude: string; velopack_icon: string; velopack_installer_base: string; velopack_mac_bundle_id: string; velopack_mac_main_exe: string; velopack_mac_pack_id: string; velopack_mac_pack_title: string; velopack_mac_pack_version: string; velopack_main_exe: string; velopack_pack_id: string; velopack_pack_title: string; velopack_pack_version: string; viewer_version: string}

(expression)